starfish/pipeline/features/codebook.py

```diff
@@ -0,0 +1,542 @@
+import functools
+import json
+from typing import Any, Dict, List, Optional, Sequence, Set
```

Comment:
is this how pycharm optimizes these imports?

Reply:
Nope, sorry. That was my brain sorting them alphabetically. Pycharm moves uuid up.